example/cumulus-tf/outputs.tf

```diff
@@ -6,17 +6,21 @@ output "archive_api_redirect_uri" {
   value = module.cumulus.archive_api_redirect_uri
 }
 
-output "distribution_url" {
-  value = module.thin_egress_app.api_endpoint
-}
+# TEA-Specific outputs. Uncomment to use TEA distribution
```

Comment:
I'm not sure I understand the point of commenting out all of the TEA code for our example deployments versus just deleting it. Whether the code is just commented or deleted, everyone's deployments will get updated to use cumulus distribution by default. And at some point carrying forward a bunch of commented code is just technical debt.

And if someone really needs/wants to use TEA, there is always git history. 

Is there something I'm missing here? Otherwise I think we just should just be deleting all this code vs commenting out

Reply:
Yup, totally valid. I had that question in my PR description here because it's a little messy. The only reason I went this way was because I could link to files locally in the README and in the moment you need to switch from one to the other, that's easier. But for all other reasons it might be a worse approach.

That said, this will be pending the outcome of the above discussions about deploying TEA AND Cognito vs. conditionally including one vs. the other. These outputs would likely also need to be conditional if we go that route.

Reply:
Jenny had a good suggestion: https://github.com/nasa/cumulus/pull/2300#discussion_r662480391. The outputs here might still need to be conditional but we could switch the other tea-only files to `xxx.tf.example` instead of commenting out OR deleting.

Reply:
>e moment you need to switch from one to the other, that's easier.

Do we need to switch from one to the other? Because we're supporting both for some time period? 

If so, then I guess going with the `example` file approach feels better than commented or deleted code. 

But yeah the outputs here would need to be conditional not commented or deleted

Reply:
The way this is configured now, the outputs in `example` are left as-is. `thin_egress` app will always deploy with `example` deployments but we can switch the distribution API we're using in main.tf. 

This was sort of the original "can we just deploy both in `example`" thought. I _think_ that seems reasonable versus having `thin_egress_app` conditional along with all of its resources and these outputs returning `null` if it's not deployed. Both are a little messy though so open to pushback.

Reply:
It's also one less step. I guess that's what pushed me to do it this way. If you want to switch between the two distribution APIs you only need to switch the assignments in main.tf. If we do something like `thin_egress_app.tf.example`, that's another step. Again, though, I'm obviously waffling 😄